File: Problems/Pure Problem/Local/Luogu/U705844/dpai/runner.py
from os import system
system("g++ base.cpp -std=c++14 -O2 -o base")
system("g++ rand.cpp -std=c++14 -O2 -o rand")
system("mkdir data")
for i in range(1, 11):
    system("bash -c \"./rand $(head -20 /dev/random | cksum | cut -c 1-10) > data.in\"")
    system("base < data.in > data.ans")
    system(f"bash -c \"cp data.in data/{i}.in\"")
    system(f"bash -c \"cp data.ans data/{i}.ans\"")

File: Problems/Pure Problem/Xes/P10483/dpai/runner.cpp
#include <windows.h>

int main() {
    system("g++ base.cpp -std=c++14 -O2 -o base");
    system("g++ rand.cpp -std=c++14 -O2 -o rand");
    system("g++ source.cpp -std=c++14 -O2 -o source");
    while (1) {
        system("bash -c \"./rand $(head -20 /dev/random | cksum | cut -c 1-10) > data.in\"");
        system("base < data.in > data.ans");
        system("source < data.in > data.out");
        int ret = system("bash -c 'diff data.out data.ans -w'");
        if (!ret) continue;
        system("pause");
    }
}